README.md

```diff
@@ -49,6 +49,55 @@ A html report of the code coverage is generated into `build/coverage/index.html`
 
 
 
+Execute marvin in docker
+--------------------------
+
+Start by creating your own copy of the configuration file.
+
+```bash
+cp marvin_config_default.json marvin_config.json
+```
+
+Edit the following settings in the file `marvin_config.json`.
```

Comment:
if you pass the config to marvin on commandline instead, you can do it in the compose file and get rid of this extra step

Reply:
That is true and ease the usage, I fix it.